components/pd_client/src/lib.rs

```diff
@@ -71,6 +73,59 @@ impl Deref for RegionInfo {
     }
 }
 
+#[derive(Default, Debug, Clone)]
+pub struct BucketMeta {
```

Comment:
Separate fields of `Buckets` here because I found only fields of `BucketMeta` need to be shared.

Reply:
I think it could make the code clear by distinguishing if we need only the meta or with stats.